lib/parsers.js

```diff
@@ -279,13 +279,32 @@ function getCulprit (frames) {
   return filename ? fnName + ' (' + filename + ')' : fnName
 }
 
+// Infer the node.js module name from the top frame filename, if possible.
+// Examples:
+//    /home/bob/.../node_modules/mymodule/index.js
+//                               ^^^^^^^^
+//    /home/bob/.../node_modules/@myorg/mymodule/index.js
+//                               ^^^^^^^^^^^^^^^
+// or on Windows:
+//    C:\Users\bob\...\node_modules\@myorg\mymodule\lib\subpath\index.js
+//                                  ^^^^^^^^^^^^^^^
+let SEP = path.sep
+if (SEP === '\\') {
+  SEP = '\\' + SEP // Escape this for use in a regex.
+}
+const MODULE_NAME_REGEX = new RegExp(`node_modules${SEP}([^${SEP}]*)(${SEP}([^${SEP}]*))?`)
 function getModule (frames) {
   if (frames.length === 0) return
   var frame = frames[0]
   if (!frame.library_frame) return
-  var match = frame.filename.match(/node_modules\/([^/]*)/)
+  var match = frame.filename.match(MODULE_NAME_REGEX)
   if (!match) return
-  return match[1]
+  var moduleName = match[1]
+  if (moduleName && moduleName[0] === '@' && match[3]) {
+    // Normalize the module name separator to '/', even on Windows.
+    moduleName += '/' + match[3]
+  }
+  return moduleName
```

Comment:
I see here we're modifying `getModule` so it works with `@namespaced/module-names` as well as with windows based filesystems. I presume this is to capture `@elastic/elasticsearch` things.

Do you think this changes requires any additional test fixtures in `test/parsers.js`, or is the existing coverage we have sufficient?

Reply:
Yah, fair, I could definitely improve the test coverage here. Actually before this change there wasn't a single test case checking `error.exception.module`. I'll open a separate ticket to do this in a followup: https://github.com/elastic/apm-agent-nodejs/issues/1918